CHANGELOG.md

```diff
@@ -21,9 +50,28 @@
 
   Please update any configurations to use `quay.io/signalfx/splunk-otel-collector:<version>` for this and future releases.
 
-### 🧰 Bug fixes 🧰
+### 💡 Enhancements 💡
+
+- (Splunk) `discovery`: Update redis discovery instructions ([#4915](https://github.com/signalfx/splunk-otel-collector/pull/4915))
+- (Splunk) `discovery`: Bring Kafkamatrics receiver into the discovery mode ([#4903](https://github.com/signalfx/splunk-otel-collector/pull/4903))
+- (Core) `mdatagen`: support setting an AttributeSet for async instruments [#9674](https://github.com/open-telemetry/opentelemetry-collector/issues/9674)
+- (Core) `mdatagen`: support using telemetry level in telemetry builder [#10234](https://github.com/open-telemetry/opentelemetry-collector/issues/10234)
+  This allows components to set the minimum level needed for them to produce telemetry. By default, this is set to configtelemetry.LevelBasic. If the telemetry level is below that minimum level, then the noop meter is used for metrics.
+- (Core) `mdatagen`: add support for bucket boundaries for histograms [#10218](https://github.com/open-telemetry/opentelemetry-collector/issues/10218)
```

Comment:
None of these things are exposed to the end user yet. There is no reason to mention changes to the metadata.yaml interface in our distro.

Reply:
removed.